<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QV_Mietvertrag</class>
 <widget class="QWidget" name="QV_Mietvertrag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mietvertrag Bearbeiten</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>480</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Als PDF abspeichern</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_paragraph">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>701</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gruppe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gruppen Paragraph</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Paragraph</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Inhalt</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="QVerwaltung.qrc">
     <normaloff>:/QVerwaltung/icons/add.png</normaloff>:/QVerwaltung/icons/add.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="QVerwaltung.qrc">
     <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>671</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Neues Element</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gruppe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gruppen-Nummer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paragraph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Inhalt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_addGroup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_addGroupNo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_addName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_addParagraph">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addPara">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hinzuf√ºgen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resetPara">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mieter:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_tennant">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FREI</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QVerwaltung.qrc"/>
 </resources>
 <connections/>
</ui>
